### PR TITLE
Bundle update and remove DB support

### PR DESCRIPTION
**Pull request by joshcanhelp**
We had a Snyk warning about a number of vulns. I updated with `bundle update` but ran into problems with `sqlite`. Since this project does not use a database, I remove ActiveRecord support as well.

### CODE REVIEW

File: 01-Authentication-RS256/config/application.rb [hunk 1]
```diff
@@ -3,14 +3,13 @@
 
 require 'rails'
 # Pick the frameworks you want:
-require 'active_model/railtie'
 require 'active_job/railtie'
-require 'active_record/railtie'
+# require 'active_record/railtie'
```
Comment (lbalmaceda) [inline 01-Authentication-RS256/config/application.rb]: any reason to keep this line?

File: 01-Authentication-RS256/config/environments/development.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@
   config.active_support.deprecation = :log
 
   # Raise an error on page load if there are pending migrations.
-  config.active_record.migration_error = :page_load
+  # config.active_record.migration_error = :page_load
```
Comment (lbalmaceda) [inline 01-Authentication-RS256/config/environments/development.rb]: Any reason to keep this line?

File: 01-Authentication-RS256/config/environments/test.rb [hunk 3]
```diff
@@ -36,7 +36,7 @@
   config.action_mailer.delivery_method = :test
 
   # Print deprecation notices to the stderr.
-  config.active_support.deprecation = :stderr
+  # config.active_support.deprecation = :stderr
```
Comment (lbalmaceda) [inline 01-Authentication-RS256/config/environments/test.rb]: Any reason to keep this line?

File: 01-Authentication-RS256/config/initializers/new_framework_defaults.rb [hunk 4]
```diff
@@ -10,7 +10,7 @@
 ActiveSupport.to_time_preserves_timezone = true
 
 # Require `belongs_to` associations by default. Previous versions had false.
-Rails.application.config.active_record.belongs_to_required_by_default = true
+# Rails.application.config.active_record.belongs_to_required_by_default = true
```
Comment (lbalmaceda) [inline 01-Authentication-RS256/config/initializers/new_framework_defaults.rb]: Any reason to keep this line?

File: 01-Authentication-RS256/test/test_helper.rb [hunk 5]
```diff
@@ -3,9 +3,9 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 
-class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all
-
-  # Add more helper methods to be used by all tests here...
-end
+# class ActiveSupport::TestCase
```
Comment (lbalmaceda) [inline 01-Authentication-RS256/test/test_helper.rb]: Any reason to keep these lines?

File: 02-Authentication-HS256/config/application.rb [hunk 6]
```diff
@@ -2,14 +2,13 @@
 
 require "rails"
 # Pick the frameworks you want:
-require "active_model/railtie"
 require "active_job/railtie"
-require "active_record/railtie"
+# require "active_record/railtie"
```
Comment (lbalmaceda) [inline 02-Authentication-HS256/config/application.rb]: Any reason to keep this line?

File: 02-Authentication-HS256/config/environments/development.rb [hunk 7]
```diff
@@ -35,7 +35,7 @@
   config.active_support.deprecation = :log
 
   # Raise an error on page load if there are pending migrations.
-  config.active_record.migration_error = :page_load
+  # config.active_record.migration_error = :page_load
```
Comment (lbalmaceda) [inline 02-Authentication-HS256/config/environments/development.rb]: Any reason to keep this line?

File: 02-Authentication-HS256/config/environments/test.rb [hunk 8]
```diff
@@ -35,7 +35,7 @@
   config.action_mailer.delivery_method = :test
 
   # Print deprecation notices to the stderr.
-  config.active_support.deprecation = :stderr
+  # config.active_support.deprecation = :stderr
```
Comment (lbalmaceda) [inline 02-Authentication-HS256/config/environments/test.rb]: Any reason to keep this line?

File: 02-Authentication-HS256/config/initializers/new_framework_defaults.rb [hunk 9]
```diff
@@ -9,7 +9,7 @@
 ActiveSupport.to_time_preserves_timezone = true
 
 # Require `belongs_to` associations by default. Previous versions had false.
-Rails.application.config.active_record.belongs_to_required_by_default = true
+# Rails.application.config.active_record.belongs_to_required_by_default = true
```
Comment (lbalmaceda) [inline 02-Authentication-HS256/config/initializers/new_framework_defaults.rb]: Any reason to keep this line?

File: 02-Authentication-HS256/test/test_helper.rb [hunk 10]
```diff
@@ -2,9 +2,9 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 
-class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all
-
-  # Add more helper methods to be used by all tests here...
-end
+# class ActiveSupport::TestCase
```
Comment (lbalmaceda) [inline 02-Authentication-HS256/test/test_helper.rb]: Any reason to keep these lines?

**Comment by joshcanhelp** [comment]
@lbalmaceda - Makes it more clear what would be different from a default Rails install. Any reason to keep them?

**Pull request merged**